.pre-commit-config.yaml

```diff
@@ -15,3 +15,8 @@ repos:
       - id: ruff
         args: [--fix, --exit-non-zero-on-fix, --show-fixes]
       - id: ruff-format
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.991
+    hooks:
+      - id: mypy
+        additional_dependencies: [types-PyYAML]
```

Comment:
I've upgraded the version, but decided not to add the `--ignore-missing-imports` and `--disallow-untyped-defs` arguments since I think those checks can be quite useful